### PR TITLE
TECH Add logstash logging stream

### PR DESCRIPTION
**Pull request by GillesRasigade**
Add support to logstash by defined 2 new environment variables:
`LOGSTASH_HOST` and `LOGSTASH_PORT`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -34,6 +36,16 @@ if (process.env.LOGENTRIES_TOKEN) {
   }));
 }
 
+if (process.env.LOGSTASH_HOST) {
+  config.streams.push({
+    type: "raw",
+    stream: logstashStream.createStream({
+      host: process.env.LOGSTASH_HOST,
+      port: process.env.LOGSTASH_PORT || 5000
```
Comment (lledey) [inline index.js]: `parseInt(process.env.LOGSTASH_PORT, 10) || 5000`

**Comment by thomasscott88** [comment]
Looks like the circleci tests fail which prevent me from approving this pull request:

/home/ubuntu/logger/index.js
41:11  error  Strings must use singlequote  quotes

Once pushed I will approve.

File: index.js [hunk 2]
```diff
@@ -34,6 +36,16 @@ if (process.env.LOGENTRIES_TOKEN) {
   }));
 }
 
+if (process.env.LOGSTASH_HOST) {
+  config.streams.push({
+    type: 'raw',
+    stream: logstashStream.createStream({
+      host: process.env.LOGSTASH_HOST,
+      port: parseInt(process.env.LOGSTASH_PORT || '5000', 10)
```
Comment (lledey) [inline index.js]: `parseInt(process.env.LOGSTASH_PORT, 10) || 5000`

File: package.json [hunk 3]
```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "bunyan": "1.8.1",
+    "bunyan-logstash": "^0.3.4",
```
Comment (lledey) [inline package.json]: Use a fixed version or `~`

File: test/index.test.js [hunk 4]
```diff
@@ -205,5 +206,24 @@ describe('index.js', () => {
       expect(config.streams).to.have.lengthOf(2);
       expect(config.streams[1].stream).to.be.instanceOf(SentryStream);
     });
+
+    it('should have logstash stream on the default port', () => {
+      process.env.LOGSTASH_HOST = 'samplehost';
```
Comment (lledey) [inline test/index.test.js]: You must reset the variable in the afterEach, otherwise if we reorder the tests they will break

Comment (lledey) [inline test/index.test.js]: It's already done, my bad

**Pull request merged**